pkgs/applications/networking/seafdav/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, python3
+, makeWrapper
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "seafdav";
+  version = "8.0.8";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seafdav";
+    rev = "v${version}-server";
```

Comment:
Upstream has a habit of pushing over existing tags, on other seafile component we end up using commit hashes.
Let's hope this is not the case here :crossed_fingers: